templates/includes/records/hierarchy-full-item.html

```diff
@@ -15,10 +15,7 @@
 {% endif %}
 
 <div class="hierarchy-full-panel__bar-container-outer">
-    <div class="hierarchy-full-panel__bar-container-inner {% if is_current_item %} hierarchy-full-panel__bar-container-inner--active {% endif %} hierarchy-full-panel__bar-container-inner--level-{{forloop.counter|add:'1'}}">
-        <p><span class="hierarchy-full-panel__bar-number">N</span>
-        <span class="hierarchy-full-panel__bar-label">record(s)</span></p>
```

Comment:
Question - Would it be helpful to retain the N records codes a commented TODO or so as we have the html tags in place.

Reply:
I think in this case having the code commented out would be useful for future development.